canvas_modules/harness/cypress/support/canvas/verification-cmds.js

```diff
@@ -868,10 +868,14 @@ Cypress.Commands.add("verifyNodeDoesExistInPaletteAtIndex", (nodeName, index) =>
 		.then((idx) => expect(idx).to.equal(index));
 });
 
-Cypress.Commands.add("verifyCategoryIsClosed", (categoryName) => {
+Cypress.Commands.add("verifyCategoryIsClosed", (categoryName) =>
+	!cy.verifyCategoryIsOpened());
+
+
+Cypress.Commands.add("verifyCategoryIsOpened", (categoryName) => {
```

Comment:
Can you use categoryLabel here as the parameter name -- since we use that in the other functions.

Reply:
Fixed.